terraform/eval_validate.go

```diff
@@ -161,7 +161,10 @@ var connectionBlockSupersetSchema = &configschema.Block{
 			Type:     cty.String,
 			Optional: true,
 		},
-
+		"target_platform": {
+			Type:     cty.String,
+			Optional: true,
+		},
```

Comment:
Since this is for SSH only, can we move it below the "For type=ssh only" comment?

(The comment claims that these settings are "enforced in ssh communicator", which I think is false, but I'd still prefer to have this new attribute in the right place.)